src/components/Modal.css

```diff
@@ -0,0 +1,44 @@
+.modal {
```

Comment:
I'd recommend adding the following style since the user cannot interact with the background when the modal is open:

```
background-color: rgb(0, 0, 0, .75);
```

Reply:
Done